### PR TITLE
feature/CWALL-121

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 1]
```diff
@@ -96,63 +108,86 @@ const SSITableView = <T extends {}>(props: Props<T>): ReactElement => {
...
       },
       ...availableColumns,
     ]
   }
+
+  const onFocusRow = (rowId?: string): void => {
+    setFocusedRowId(rowId)
+  }
+
+  const onRowSelectionChange: OnChangeFn<RowSelectionState> = (updatedRowSelection): void => {
```
Comment (nklomp) [inline packages/ssi-react/src/components/views/SSITableView/index.tsx:164]: Can't we type this argument?

I am assuming that is also why the next ts-ignore is present.

File: packages/ssi-react/src/components/views/SSITableView/index.tsx [hunk 2]
```diff
@@ -96,63 +108,86 @@ const SSITableView = <T extends {}>(props: Props<T>): ReactElement => {
...
+    // @ts-ignore
+    const currentRowSelection = updatedRowSelection(toRowSelectionObject(rowSelection))
+
+    const selection: Array<TableRowSelection> = []
+    for (const key in currentRowSelection) {
```
Comment (nklomp) [inline packages/ssi-react/src/components/views/SSITableView/index.tsx:169]: Is there a reason for a traditional loop? Could be quite easily mapped from the looks of it

**Review by nklomp — CHANGES_REQUESTED** [review]
Some small remarks, questions

File: packages/ssi-react/src/components/views/SSITableView/index.tsx (hunk 1 above)
Comment (BtencateSphereon) [reply packages/ssi-react/src/components/views/SSITableView/index.tsx]: Added type and a comment. It's complaining about something i have no solution for yet and did not want to spend more time on it

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**